src/test/java/core/basesyntax/service/impl/WriteServiceImplTest.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.ReadService;
+import core.basesyntax.service.WriteService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class WriteServiceImplTest {
+    private WriteService writeService;
+    private ReadService readService;
+    private Map<String, Integer> testMap;
+
+    @Before
+    public void beforeEachTest() {
+        writeService = new WriteServiceImpl();
+        readService = new ReadServiceImpl();
```

Comment:
your [services should be tested independently](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop-tests#ensure-that-you-test-your-services-and-their-independently), pay attention to it next time